leveldb/leveldb_test.go

```
@@ -0,0 +1,166 @@
+	require.NotNil(t, db)
+	_, err := os.Stat("./testdb")
+	require.Nil(t, err)
+
+	err = removeAllFiles("./testdb")
+	require.Nil(t, err)
+}
+
+func Test_Set_Overwrite(t *testing.T) {
+	db := New()
+
+	db.Set([]byte("key"), []byte("value"), time.Second*1)
```

golangci-lint (1.62.2)

Error return value of `db.Set` is not checked

(errcheck)